HameedKareem.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Hameed Kareem",
+  "bio": "A Self-Thought DevOps Engineer Who is Passionate in Delivering Software Products in high velocity by Designing Solutions that helps businesses grows.",
+    "tags": ["Linux", "Python", "CI/CD","Docker", "Kubernetes"],
+
+   "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/hameed__kareem"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/maxton4life"
+    },
+    {
+      "icon": "SiHashnode",
+      "url": "https://hashnode.com/@hameedkareem"
+    }
+  ],
+  "links": [
+    {
+      "name": "Connect with me on Twitter",
+      "url": "https://twitter.com/hameed__kareem",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Connect with me on github",
+      "url": "https://github.com/maxton4life",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/abdulhameed-abdulkareem123",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Open Source Blog",
+      "url": "https://hashnode.com/@hameedkareem",
+      "icon": "SiHashnode"
+    }
+  ]
+}
+  
+    }
```

Comment:
Please remove this extra closing curly brace.

Reply:
How do I access the file cant seem to edit it?

Reply:
![image](https://github.com/EddieHubCommunity/BioDrop/assets/93700622/f9c598b0-0dbf-4e7a-a53f-8ef2cea3c89f)
Click on edit file and then you can edit the file.

Reply:
I can't find seem to find the file on my forked. It's not in the main branch.